test/segment-loader.test.js

```diff
@@ -451,18 +451,16 @@ QUnit.test('adjusts the playlist offset if no buffering progress is made after'
   sourceBuffer.buffered = videojs.createTimeRanges([[0, 10]]);
   sourceBuffer.trigger('updateend');
 
-  for (let i = 1; i <= 6; i++) {
+  for (let i = 1; i <= 5; i++) {
     // the next segment doesn't increase the buffer at all
     QUnit.equal(this.requests[0].url, (i + '.ts'), 'requested the next segment');
     this.clock.tick(1);
     this.requests[0].response = new Uint8Array(10).buffer;
     this.requests.shift().respond(200, null, '');
     sourceBuffer.trigger('updateend');
   }
-
-  // so the loader should try the next segment
-  QUnit.equal(errors, 1, 'emitted error');
-  QUnit.ok(loader.paused(), 'loader is paused');
```

Comment:
It's a little weird that the loader is not paused but not fetching segments. I understand we don't want to trigger blacklisting in this particular case but maybe we should trigger a non-error event and pause the loader?